coalib/bearlib/languages/definitions/SCSS.py

```diff
@@ -0,0 +1,48 @@
+from {
+	coalib.bearlib.languages.Language {
+		import {
+			Language {
+				@Language {
+					class {
+						CSS {
+							&: {
+								extensions {
+									= {
+										'.css' {
+											'/*': '*/';
+										}
+									}
+								}
+							}
+						}
+					}
+				}
+			}
+		}
+	}
+}
+multiline_comment_delimiters {
+	= {
+		'/*': '*/';
+	}
+}
+string_delimiters {
+	= {
+		'"': '"', "'": "'";
+	}
+}
+indent_types {
+	= {
+		'{': '}';
+	}
+}
+encapsulators {
+	= {
+		'(': ')', '[': ']';
+	}
+}
+string_delimiter_escape {
+	= {
+		'"': '\\"', "'": "\\'";
+	}
```

Comment:
You need to change the formatting of the whole file. It's not as we expect it to be.

Reply:
This file is not in python at all